pandas/tests/frame/test_combine_concat.py

```diff
@@ -129,6 +129,17 @@ def test_concat_tuple_keys(self):
         )
         assert_frame_equal(results, expected)
 
+    def test_append_empty_list(self):
```

Comment:
can you parameterize with empty list, tuple, np.ndarray

Reply:
@jreback Please see the reply in the other comment (https://github.com/pandas-dev/pandas/pull/28834#discussion_r334256800)